### ISSUE TITLE
labels for contexts that are listed in the toolwindow are hardly readable

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-kubernetes#42]
depends on #34 

the contexts that are listed in the tool window are very long. They currently match the way that's used in vscode which is to display `<current namespace>/<host>/<user>`

![image](https://user-images.githubusercontent.com/25126/82607389-f8485880-9bb8-11ea-8665-c32910e6f785.png)

A possible improvement could be to display the cluster and decorate it namespace & user with lighter letters like one can see here within the projects view:

![image](https://user-images.githubusercontent.com/25126/82607501-36457c80-9bb9-11ea-98be-537151e47e0d.png)

**Comment by adietish** [comment]
Apparently those labels are being discussed in vscode. I'll extract the suggested change for these labels on hold until we have a decision in vscode and try to match what they'll provide.
